<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SUNNY-HP" name="Command line test" tests="5" failures="1" timestamp="20 Aug 2015 18:04:53 GMT" time="3.137" errors="0">
  <testcase name="getJobs" time="0.191" classname="csjobs.model.dao.JobDaoTest">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
	at csjobs.model.dao.JobDaoTest.getJobs(JobDaoTest.java:26)
	at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.run(AbstractTestNGSpringContextTests.java:171)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- getJobs -->
  <testcase name="getUserByEmail" time="0.057" classname="csjobs.model.dao.UserDaoTest"/>
  <testcase name="getUserById" time="0.004" classname="csjobs.model.dao.UserDaoTest"/>
  <testcase name="checkApplication" time="0.068" classname="csjobs.model.dao.UserDaoTest"/>
  <testcase name="countCommittees" time="0.0" classname="csjobs.model.dao.JobDaoTest">
    <skipped/>
  </testcase> <!-- countCommittees -->
</testsuite> <!-- Command line test -->
